00-Introducing-HOOMD-blue/02-Performing-Hard-Particle-Monte-Carlo-Simulations.ipynb

```diff
@@ -0,0 +1,209 @@
+{
```

Comment:
<p>It is a bit pedantic but I may use results rather than trajectory in the first usage in case people aren't familiar with the term.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>done.</p>